fang/src/lib.rs

```diff
@@ -158,6 +179,68 @@ pub struct Task {
     pub updated_at: DateTime<Utc>,
 }
 
+#[cfg(feature = "asynk-sqlx")]
+impl<'a> FromRow<'a, AnyRow> for Task {
+    fn from_row(row: &'a AnyRow) -> Result<Self, sqlx::Error> {
+        let uuid_as_text: &str = row.get("id");
+
+        let id = Uuid::parse_str(uuid_as_text).unwrap();
+
+        let raw: &str = row.get("metadata"); // will work if database cast json to string
+        let raw = raw.replace('\\', "");
+
+        // -- SELECT metadata->>'type' FROM fang_tasks ; this works because jsonb casting
+        let metadata: serde_json::Value = serde_json::from_str(&raw).unwrap();
+
+        // This should be changed when issue https://github.com/launchbadge/sqlx/issues/2416 is fixed
+        // Fixed in pxp9's fork
```

Comment:
probably these comments can be removed

Reply:
[deleting prints and updating comments](https://github.com/ayrat555/fang/pull/141/commits/6f85163f5fec7ebb098d9c5493f650c6b864aed4)